e2e/test/scenarios/embedding/embedding-hub/embedding-hub.cy.spec.ts

```diff
@@ -110,17 +110,64 @@ describe("scenarios - embedding hub", () => {
         .should("be.visible");
     });
 
-    it('"Create models" link should navigate correctly', () => {
+    it('"Create models" step should be marked as done after creating a model', () => {
+      cy.intercept("POST", "/api/dataset").as("dataset");
+
+      H.addSqliteDatabase("Test Database");
+
       cy.visit("/admin/embedding/setup-guide");
 
-      cy.log("Find and click on 'Create models' link");
+      cy.log("'Create models' should not be marked as done initially");
+      cy.findByTestId("admin-layout-content")
+        .findByText("Create models")
+        .closest("button")
+        .findByText("Done")
+        .should("not.exist");
+
       cy.findByTestId("admin-layout-content")
         .findByText("Create models")
         .first()
         .click();
 
-      cy.log("Should navigate to model creation page");
       cy.url().should("include", "/model/new");
+
+      cy.log("Create a native query model");
+      cy.findByTestId("new-model-options")
+        .findByText("Use a native query")
+        .click();
+
+      cy.log("Use the test database");
+      H.popover().findByText("Test Database").click();
+
+      H.NativeEditor.focus().type("SELECT 1 as t");
```

Comment:
Creating a model via the UI isn't a part of our concern, unless you wanna ensure the native query builder works.

I'd suggest you create a model via the API to speed up this test.